### PR TITLE
[Experimental] Update to JRE 11

### PR DESCRIPTION
**Pull request by idelcano**
test docker with jre-11

tomcat have changed the base OS by debian.

Notes:
pipefail doesn't work.
I have commented the packages offline install.

### CODE REVIEW

File: src/d2_docker/config/dhis2-core-start.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -x
-set -e -u -o pipefail
+set -e -u 
+#-o pipefail
```
Comment (SferaDev) [inline src/d2_docker/config/dhis2-core-start.sh:3]:
```suggestion
```

File: src/d2_docker/config/server.xml [hunk 2]
```diff
@@ -31,7 +31,8 @@
             protocol="AJP/1.3"
             port="8009"
             redirectPort="8443"
-        />
+	    secretRequired="false"
+	    />
```
Comment (SferaDev) [inline src/d2_docker/config/server.xml:35]:
```suggestion
	/>
```

File: src/d2_docker/config/dhis2-core-entrypoint.sh [hunk 3]
```diff
@@ -6,10 +6,14 @@
 #  - Make files in TOMCATDIR group-writable (so we can change tomcat files in pre/post scripts)
 #  - Install some dependencies: curl, postgresql-client
 #
-set -e -u -o pipefail
+SHELL=/bin/bash
+set -e -u 
+#-o 
+#| grep pipefail
+#set -e -u -o pipefail
```
Comment (SferaDev) [inline src/d2_docker/config/dhis2-core-entrypoint.sh:13]:
```suggestion
```

**Comment by tokland** [comment]
Maybe change default for postgres to 13

**Comment by jordibc** [comment]
I think the last commit fixed it.

**Comment by idelcano** [comment]
I got the following error:
ERROR: for d2-docker-docker-eyeseetea-com-widp-2-38-test_data_1  Cannot start service data: failed to create shim: OCI runtime create failed: runc create failed: unable to start container process: exec:

**Comment by jordibc** [comment]
> 

Thanks! There was a problem with the data image indeed. The last commit fixes it.

**Pull request merged**